src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -27,36 +32,51 @@ fun Class<*>.isKotlinClass(): Boolean {
  *                                      (e.g. List<String>) may contain null values after deserialization.  Enabling it
  *                                      protects against this but has significant performance impact.
  */
-class KotlinModule constructor (
-    val reflectionCacheSize: Int = 512,
-    val nullToEmptyCollection: Boolean = false,
-    val nullToEmptyMap: Boolean = false,
-    val nullIsSameAsDefault: Boolean = false,
-    val singletonSupport: SingletonSupport = DISABLED,
-    val strictNullChecks: Boolean = false
+class KotlinModule private constructor(
```

Comment:
Is this backwards compatible change wrt sources? I am not super familiar with Kotlin visibility/access levels so I may be wrong but reducing visibility seems like it wouldn't be.

Reply:
It should be now that japicmp isn't failing the build.